tox.ini

```diff
@@ -37,15 +37,14 @@ deps =
     mccabe
     pep8
     pep8-naming
-    pycodestyle<2.4.0
```

Comment:
This constraint was causing an error between flake8 and pycodestyle. I just removed it since the latest release for both work fine together and nothing else had constraints.